.github/workflows/ci.yml

```diff
@@ -292,3 +389,27 @@ jobs:
           name: playwright-traces-${{ github.run_id }}-${{ github.run_attempt }}-${{ matrix.test-name }}
           path: integration/test-results
           retention-days: 1
+
+  pkg-pr-new:
+    name: Publish with pkg-pr-new
+    if: ${{ github.event_name != 'pull_request' || github.event.pull_request.draft == false }}
+    needs: [check-permissions, build-packages]
+    runs-on: 'blacksmith-8vcpu-ubuntu-2204'
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          show-progress: false
+
+      - name: Setup Node
+        uses: ./.github/actions/init-blacksmith
+        with:
+          turbo-enabled: true
+          node-version: 22
+
+      - name: Publish with pkg-pr-new
+        run: pnpm run build && pnpx pkg-pr-new publish --compact --pnpm './packages/*'
```

Comment:
❓ is there any benefit in using `pnpx` here instead of installing `pkg-pr-new` in the root package.json?

Reply:
BTW, do want this to run for each commit or should we trigger it by commenting `!snapshot` (essentially replacing the prev snapshot impl)

Reply:
> ❓ is there any benefit in using `pnpx` here instead of installing `pkg-pr-new` in the root package.json?

It allows us not to carry `pkg-pr-new` as a dependency that gets installed all the time and requires upkeep. It's only used in its own CI workflow so the delay introduced by pnpx isn't really an issue

Reply:
> BTW, do want this to run for each commit or should we trigger it by commenting `!snapshot` (essentially replacing the prev snapshot impl)

@nikosdouvlis That's a good question. Their docs market it as Continuous Releases, so they point you to running it on all PR commits. It could be overkill and a bit wasteful of CI resources if we don't really use these releases a lot. A comment or PR label as opt-in might be better. @clerk/sdk-infra thoughts?